site/components/PageNumbers.tsx

```diff
@@ -0,0 +1,292 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import { Dispatch, ReactElement, SetStateAction } from "react";
+
+interface PageNumbersProps {
+    currentPage: number;
+    setCurrentPage: Dispatch<SetStateAction<number>>;
+    numberOfPages: number;
+}
+
+const createPageNumbers = (
+    numberOfPages: number,
+    currentPage: number,
+    setCurrentPage: Dispatch<SetStateAction<number>>,
+) => {
+    if (numberOfPages < 10) {
+        const pageNumbers = [];
+
+        for (let count = 0; count < numberOfPages; count++) {
+            pageNumbers.push(
+                <li
+                    key={`page-${count + 1}`}
+                    className={`govuk-pagination__item${
+                        count + 1 === currentPage ? " govuk-pagination__item--current" : ""
+                    }`}
+                >
+                    <a
```

Comment:
Next recommends using the <Link> component over <a> tags, which has the added benefit of being able to remove the eslint disable

Reply:
Done